### PR TITLE
Make the banner scrollable on 200% text size

### PR DESCRIPTION
**Pull request by AmelBawa-msft**
## References and relevant issues

Work item number: 45063344

## PR checklist
- [ ] Closes #xxx
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

File: common/Views/Banner.xaml [hunk 1]
```diff
@@ -6,39 +6,41 @@
...
         </Grid.Background>
 
-        <!-- Set Min sizes so that image gets clipped as window narrows, rather than scaling down -->
-        <Image Stretch="Uniform" Source="{x:Bind OverlaySource, Mode=OneWay}" 
+        <!-- Background image -->
+        <Image Height="{Binding ElementName=BannerGrid, Path=Height}"
+               Stretch="Uniform" Source="{x:Bind OverlaySource, Mode=OneWay}"
                AutomationProperties.AccessibilityView="Raw"
-               Margin="0 0 0 0" MinHeight="214" MinWidth="{ThemeResource MaxPageContentWidth}"
+               Margin="0 0 0 0" MinWidth="{ThemeResource MaxPageContentWidth}"
```
Comment (dhoehna) [inline common/Views/Banner.xaml:18]: margin of all zeros?

File: common/Views/Banner.xaml [hunk 2]
```diff
@@ -6,39 +6,41 @@
...
+            <StackPanel VerticalAlignment="Center" Margin="38 0 0 0" Spacing="12">
+                <TextBlock MaxWidth="{x:Bind TextWidth}"
+                           HorizontalAlignment="Left"
+                           Style="{ThemeResource SubtitleTextBlockStyle}"
+                           Text="{x:Bind Title}"/>
+                <TextBlock MaxWidth="{x:Bind TextWidth}"
+                           HorizontalAlignment="Left"
+                           TextWrapping="Wrap"
+                           Text="{x:Bind Description}"/>
+                <Button Margin="0 15 0 0"
```
Comment (dhoehna) [inline common/Views/Banner.xaml:32]: I understand the margin.  Why the padding though?  Excessive padding tells me a stack panel isn't the correct container.

**Review by dhoehna — APPROVED** [review]

File: common/Views/Banner.xaml (hunk 1 above)
Comment (AmelBawa-msft) [reply common/Views/Banner.xaml]: Removed/Not needed.

File: common/Views/Banner.xaml (hunk 2 above)
Comment (AmelBawa-msft) [reply common/Views/Banner.xaml:32]: The padding is relative to the Button element not the container. This determines how much space is between the button text and the button border. Let me know if you have other concerns.

image

**Review by bbonaby — APPROVED** [review]

**Pull request merged**